pandas/io/formats/style_render.py

```diff
@@ -1433,7 +1548,7 @@ def _wrap_decimal_thousands(
     """
 
     def wrapper(x):
-        if isinstance(x, (float, complex, int)):
+        if isinstance(x, (float, complex)) or is_integer(x):
```

Comment:
For consistency you should use `is_float` and `is_complex` which are looser (e.g., allowing NumPy float64)

Reply:
- [x] chged